zombie_dice/ai.go

```diff
@@ -0,0 +1,24 @@
+package zombie_dice
+
+import (
+	"github.com/Akavall/GoGamesProject/dice"
+)
+
+func GreedyAI(shots int) int {
```

Comment:
I think these are perfect candidates for an AI interface! They can have their own name but all implement a "shouldKeepGoing()" method (or something similar).